.github/workflows/ci.yaml

```diff
@@ -13,7 +13,7 @@ env:
 
 jobs:
   lint:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
bumping to 24.04 to get the sgx and dcap versions

While we might consider updating this to be n'sync with the main mobilecoin repo it may not be worth the effort right now